### PR TITLE
changes in axops to talk to any s3 compatible object store

### LINKED ISSUE #244: changes in axops to talk to any s3 compatible object store

### PR DESCRIPTION
**Pull request by sandeepbhojwani**
Resolves #244

### CODE REVIEW

File: saas/axops/src/applatix.io/axops/axops_server/main.go [hunk 1]
```diff
@@ -65,14 +65,14 @@ func main() {
 
 	if axErr := axops.CreateDomainManagementTool(); axErr != nil {
 		utils.ErrorLog.Println("Failed to load domain configuartions:", axErr)
-		os.Exit(1)
+		//os.Exit(1)
```
Comment (shrinandj) [inline saas/axops/src/applatix.io/axops/axops_server/main.go]: Should this be uncommented?

File: saas/common/src/applatix.io/s3cl/s3cl.go [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/credentials"
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/aws/aws-sdk-go/service/s3"
+	"net/http"
+	"os"
+)
+
+var s3Region, _ = os.LookupEnv("AX_REGION")
+var s3EndPoint, _ = os.LookupEnv("ARGO_S3_ENDPOINT")
```
Comment (shrinandj) [inline saas/common/src/applatix.io/s3cl/s3cl.go]: What happens if this env variable is not set? Is there a way to set some defaults?

Comment (sandeepbhojwani) [reply saas/common/src/applatix.io/s3cl/s3cl.go]: variables will be empty strings if env variables are not set. If s3Endpoint is empty i assume we are in aws and use the aws specific way of creating session.

File: saas/axops/src/applatix.io/axops/axops_server/main.go (hunk 1 above)
Comment (sandeepbhojwani) [reply saas/axops/src/applatix.io/axops/axops_server/main.go]: yes, good catch

**Pull request merged**